utils/merkletrie/fsnoder/file.go

```diff
@@ -0,0 +1,73 @@
+package fsnoder
+
+import (
+	"bytes"
+	"fmt"
+	"hash/fnv"
+
+	"gopkg.in/src-d/go-git.v4/utils/merkletrie/noder"
```

Comment:
Having a noder subpackage seems overdoing it, since noder is not meant to be used independently of merkletrees.